src/backend/api/wine.ts

```diff
@@ -29,3 +31,7 @@ export const handleProgressOfWinetricks = (onProgress: any) =>
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const progressOfWinetricksRemoveListener = (onProgress: any) =>
   ipcRenderer.removeListener('progressOfWinetricks', onProgress)
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const handleProgressOfWineManager = (version: string, callback: any) =>
+  ipcRenderer.on('progressOfWineManager' + version, callback)
```

Comment:
```suggestion
export const handleProgressOfWineManager = (version: string, onProgress: { state: State, progress: ProgressInfo}) =>
  ipcRenderer.on('progressOfWineManager' + version, onProgress)
```

Reply:
this should be 
```
export const handleProgressOfWineManager = (
  version: string,
  callback: (
    e: Electron.IpcRendererEvent,
    progress: {
      state: State
      progress: ProgressInfo
    }
  ) => void
) => {
  ipcRenderer.on('progressOfWineManager' + version, callback)
  return () => {
    ipcRenderer.removeListener('progressOfWineManager' + version, callback)
  }
}
```
This is reflected in the latest commit.